compclub/website/urls.py

```diff
@@ -1,10 +1,14 @@
-from django.urls import include, path
+from django.conf.urls import url
+from django.contrib.auth import views as auth_views
+from django.urls import path
 
 from . import views
 
 urlpatterns = [
+    url(r'^login/$', auth_views.login, {'template_name': 'website/login.html'}, name='login'),
```

Comment:
I notice that we're going to a URL with no template once we're logged in. As a stop-gap measure, let's redirect the user to `index` when login is successful.